packages/tocco-ui/src/Menu/Item.js

```diff
@@ -0,0 +1,66 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+const ItemStyles = styled.li`
+  position: relative;
+
+  > ul {
+    display: ${props => props.isOpen ? 'block' : 'none'}
+  }
+`
+
+class Item extends React.Component {
+  state = {
+    isOpen: this.props.isOpen
+  }
+
+  toogleOpenState = () => {
+    // eslint-disable-next-line
+    this.props.isToggable
+    && this.setState(prevState => ({
+      isOpen: !prevState.isOpen
+    }))
+  }
+
+  childs // eslint-disable-next-line
+  = React.Children.map(this.props.children, child => {
+    return React.cloneElement(child, {look: this.props.look})
+  })
```

Comment:
- This should be moved into the `render` function. Otherwise the props would be outdated after the first update. The same `Item` instance can be reused over time; updated with new props.
- The plural word for "child" is "children" (not "childs")

Reply:
I factored children out of render() to increase reusability (DRY). If variable children is wrapped into a method is it still get outdated? What do you think about following proposal?

  getChildren = () => {
    // eslint-disable-next-line
    return React.Children.map(this.props.children, child => {
      return React.cloneElement(child, {look: this.props.look})
    })
  }

  render() {
    return (
      <ItemStyles
        isOpen={this.state.isOpen}
        isToggable={this.props.isToggable}
      >
        {this.getChildren()}
      </ItemStyles>
    )
  }

Reply:
see 87d87670dbeb51c2609d4a43efc9e8e487a6c1ee